drivers/adc-dac/ad5592r/ad5592r.h

```diff
@@ -49,6 +49,8 @@
 	((((x) & 0x00ff) << 8) | \
 	 (((x) & 0xff00) >> 8))
 
+extern const struct ad5592r_rw_ops ad5592r_rw_ops;
```

Comment:
why would this need to be shared with anything else? The driver is the only one that would use these operations.